app/src/main/java/com/codepath/earthquakemonitor/models/Earthquake.java

```diff
@@ -0,0 +1,228 @@
+package com.codepath.earthquakemonitor.models;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Created by emilie on 10/9/17.
+ */
+public class Earthquake {
+
+    //todo clean up the unnecessary param
+    private Double mag;
+    private String place;
+    private Integer time;
+    private String url;
+
+    private Double longitude;
+    private Double latitude;
+    private Double depth;
+
+
+
+    public Double getMag() {
+        return mag;
+    }
+    public void setMag(Double mag) {
+        this.mag = mag;
+    }
+    public Earthquake withMag(Double mag) {
+        this.mag = mag;
+        return this;
+    }
+    public String getPlace() {
+        return place;
+    }
+    public void setPlace(String place) {
+        this.place = place;
+    }
+    public Earthquake withPlace(String place) {
+        this.place = place;
+        return this;
+    }
+    public Integer getTime() {
+        return time;
+    }
+    public void setTime(Integer time) {
+        this.time = time;
+    }
+    public Earthquake withTime(Integer time) {
+        this.time = time;
+        return this;
+    }
+    public String getUrl() {
+        return url;
+    }
+    public void setUrl(String url) {
+        this.url = url;
+    }
+
+    public Double getLongitude() {
+        return longitude;
+    }
+    public void setLongitude(Double longitude) {
+        this.longitude = longitude;
+    }
+
+    public Double getLatitude() {
+        return latitude;
+    }
+    public void setLatitude(Double latitude) {
+        this.latitude = latitude;
+    }
+
+    public Double getDepth() {
+        return depth;
+    }
+    public void setDepth(Double depth) {
+        this.depth = depth;
+    }
+
+
+    public static Earthquake fromJson(JSONObject jsonObject)throws JSONException{
+        Earthquake earthquake = new Earthquake();
+
+        JSONObject properties = jsonObject.getJSONObject("properties");
+
+        earthquake.setMag(properties.getDouble("mag"));
+        earthquake.setPlace(properties.getString("place"));
+        earthquake.setTime(properties.getInt("time"));
+        earthquake.setUrl(properties.getString("url"));
+
+        JSONArray coordinates = jsonObject.getJSONObject("geometry").getJSONArray("coordinates");
+
+        earthquake.setLongitude(coordinates.getDouble(0));
+        earthquake.setLatitude(coordinates.getDouble(1));
+        earthquake.setDepth(coordinates.getDouble(2));
+
+        return earthquake;
+    }
+
+    //Those commented param might be interesting
```

Comment:
We can get a loooooot of params from the earthquake API. I just pasted them in this file so you can have a look in case you find one interesting.